mm2src/mm2_main/src/lp_swap.rs

```diff
@@ -258,9 +261,7 @@ pub async fn process_msg(ctx: MmArc, topic: &str, msg: &[u8]) {
                     error!("Couldn't deserialize 'SwapStatus': {:?}", swap_status_err);
                 },
             };
-            // Drop it to avoid dead_code warning
-            drop(swap_msg_err);
-            return;
+            return MmError::err(P2PRequestError::DecodeError(swap_msg_err.to_string()));
```

Comment:
the same

Reply:
it's logged up above already

Reply:
That is why it does not require to return `MmError`. 

```rust
if lp_swap::process_msg(ctx.clone(), split.next().unwrap_or_default(), &message.data)
    .await
    .is_err()
{
    return;
}
```

Reply:
This is not from latest branch:

https://github.com/KomodoPlatform/atomicDEX-API/blob/911bc31a1aa96cb9cd7e5d3dee179b26ded93c78/mm2src/mm2_main/src/lp_network.rs#L213-L216